migrations/20240513163957_create-offer-table.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Migration for creating 'offer' table.
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('offer', (table) => {
+    table.increments('Id').primary().notNullable()
+    table.dateTime('SentAt')
+    table.dateTime('ExpiresAt').notNullable()
+    table.dateTime('AnsweredAt')
+    table.string('SelectionSnapshot').notNullable() // JSON

```

Comment:
Det finns en json-typ som jag tycker vi borde använda istället för string

Reply:
Jag tänkte också det först men jag tror inte det finns en JSON typ i sql server, även om knex exponerar den.

https://learn.microsoft.com/en-us/sql/relational-databases/json/store-json-documents-in-sql-tables?view=sql-server-ver16#json-storage-format

Reply:
Synd! Då kör vi på string 💾